src/components/simulation/LargeFooter.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts">
+import { UiRennesLogo } from '@sigrennesmetropole/cooperation_jn_common_ui'
+import { UiLinkFooter } from '@sigrennesmetropole/cooperation_jn_common_ui'
+import type { LegalItem } from '@/model/legalItem.model'
+import type { PropType } from 'vue'
+
+const openLink = (link: string) => {
+  window.open(link, '_blank')
+}
+
+const props = defineProps({
+  textColor: {
+    type: String,
+    required: false,
+    default: 'neutral-500',
+  },
+  legalList: Array as PropType<LegalItem[]>,
+})
+</script>
+
+<template>
+  <footer class="flex flex-row justify-between font-dm-sans" id="footer">
+    <div class="flex p-0 gap-6">
+      <UiRennesLogo class="w-36 h-7"></UiRennesLogo>
+      <div class="font-normal text-xs">
+        Visualisez les données thématiques <br />de notre territoire sur sa
```

Comment:
I think the text is not update on figma but the new text is :"Visualisez les données thématiques du territoire sur la maquette 3D"